ndex2/client.py

```diff
@@ -391,6 +437,18 @@ def save_cx_stream_as_new_network(self, cx_stream, visibility=None):
     def update_cx_network(self, cx_stream, network_id):
         """
         Update the network specified by UUID network_id using the CX stream cx_stream.
+	
+	.. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		cx_stream = 'INSERT CX STREAM HERE'
+		network_id = 'INSERT NETWORK UUID HERE'
+		my_ndex.update_cx_network(cx_stream, network_id)
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client